src/ui/pages/BridgeForm/BridgeForm.tsx

```diff
@@ -104,6 +108,18 @@ import { SuccessState } from './SuccessState';
 import { LabeledNetworkSelect } from './LabeledNetworkSelect';
 import { BridgeLine } from './BridgeLine';
 import { ZerionFeeLine } from './ZerionFeeLine';
+import { QuoteList } from './QuoteList';
+
+function getDefaultQuote(quotes: Quote[] | null) {
+  return quotes?.[0] ?? null;
+}
+
+function useSortedQuotes(params: Parameters<typeof useQuotes>[0]) {
+  return {
+    quotesByAmount: useQuotes({ ...params, sortType: 'amount' }),
```

Comment:
Not sure about this double request to be honest?
Do we really need this? In 99% cases users will not try to change the quote